src/components/ProfilePage.jsx

```diff
@@ -201,7 +201,7 @@ ProfilePage.defaultProps = {
 const mapStateToProps = (state) => {
   const profileImage =
     state.profilePage.account.profileImage != null
-      ? state.profilePage.account.profileImage.imageUrlLarge
+      ? state.profilePage.account.profileImage.image_url_large
```

Comment:
It would be cool, if we ran `camelcaseKeys` with the `deep: true` option set down at the service layer instead, but this works.

Reply:
Yea I think David has work that addresses this.

Reply:
This will be fixed on my current branch, yeah.  I cleaned up alllll the various transform/case changing code.